core/api/test/unit/domain/users/phone-metadata-authorizer.spec.ts

```diff
@@ -154,14 +154,14 @@ describe("PhoneMetadataAuthorizer - validate", () => {
 
   it("returns error with undefined metadata", () => {
     const validator = PhoneMetadataAuthorizer(
-      getPhoneMetadataRewardsSettings(),
+      getPhoneMetadataQuizzesSettings(),
     ).authorize(undefined)
     expect(validator).toBeInstanceOf(ExpectedPhoneMetadataMissingError)
   })
 
   it("returns error with voip type", () => {
     const validator = PhoneMetadataAuthorizer(
-      getPhoneMetadataRewardsSettings(),
+      getPhoneMetadataQuizzesSettings(),
```

Comment:
quizzes or quiz?

Reply:
it depend of the context on whether it's singular or plural